untl/README.rst

```diff
@@ -0,0 +1,431 @@
+:Comment: This element can be used to describe the original (physical or digital) resource from which the current digital resource is derived. The present resource may be derived from the source resource in whole or in part. 
+:Obligation: Not required
+:Repeatability: Repeatable
+:Data Type: String
+:Data Constraint: None
+:Qualifier: http://purl.org/NET/UNTL/vocabularies/sourceQualifiers/
+
+Relation
+--------
+
+:Element: relation
+:Label: Relation
+:Sub Elements: None
+:Definition:  Information about another resource that is related to the current resource.  It includes an expression of the relationship type.
+:Comment:  Relation is simply a reference to a related resource. It is essential to maintaining a history of the change of a related digital resource. It specifies any other information resources, which were judged to be significantly related to the digital resource being described and necessary for preservation management. It also enables a digital resource to be linked to earlier or later versions/editions of it, other forms of it, to its metadata, and other objects, including finding aids. 
+:Obligation: Not required
+:Repeatability: Repeatable
+:Data Type: String
+:Data Constraint: None
+:Qualifier: http://purl.org/NET/UNTL/vocabularies/relation-qualifiers/
+
+
+Collection
+----------
+
+:Element: collection
+:Label: Collection
+:Sub Elements: None
+:Definition: Collection name refers to a larger group of resources with a unique collective title to which the resource being described belongs.
+:Comment:  Use the drop-down list to select a controlled collection name.
+:Obligation: Not required
+:Repeatability: Repeatable
+:Data Type: String
+:Data Constraint: http://purl.org/NET/UNTL/vocabularies/collections/
+:Qualifier: None
+
+Institution
+-----------
+
+:Element: institution
+:Label: Institution
+:Sub Elements: None
+:Definition: A consistent reference to the institution or administrative unit that owns the digital resource for which metadata was created.
+:Comment: Use the drop-down list to select a controlled  institution name.
+:Obligation: Not required
+:Repeatability: Repeatable
+:Data Type: String
+:Data Constraint: http://purl.org/NET/UNTL/vocabularies/institutions/
+:Qualifier: None
+
+
+Rights
+------
+:Element: rights
+:Label: Rights
+:Sub Elements: [None] 
+:Definition: Rights information provides complet information about rights held in and over the resource describes the conditions under which the work may be used, distributed, reproduced, etc., how these conditions may change over time, and whom to contact regarding the copyright of the work.
+:Comment: Access describes the level of access that the resource is allowed.License reprenets rights ot license associated with the resources. Rights holder is the one who holds the license/rights to the resources. And rights statements provides statement or a link to a detailed statement of the rights for the resource.
```

Comment:
Add space in "allowed.License".  Also, there are typos in "reprenets rights ot". Plus, sentence shouldn't start with "And" . Make "rights statements" singular.